src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -30,5 +30,14 @@ public static void EnableSetFunction(this PowerFxConfig powerFxConfig)
         {
             powerFxConfig.AddFunction(new RecalcEngineSetFunction());
         }
+
+        /// <summary>
+        /// Enable a Collect() function which allows scripts to append table records.
+        /// </summary>
+        /// <param name="powerFxConfig"></param>
+        public static void EnableCollectFunction(this PowerFxConfig powerFxConfig)
```

Comment:
>this PowerFxConfig powerFxConfig [](http://example.com/codeflow?start=49&length=32)

Update: add to this SymbolTable symbolTable, instead

Reply:
Done.